portfolio/src/main/webapp/index.html

```diff
@@ -53,19 +53,25 @@ <h2>About</h2>
                     </div>
                 </div>
 
-                <div class="row align-middle center">
-                     <form action="/comments" method="POST">
+                <div class="row align-middle">
+                     <form action="/post-comment" method="POST">
                         <p>Enter Your Comment Here</p>
                         <label>User Name: </label><input type="text" name="user-name"><br/>
                         <label>Email Address: </label><input type="text" name="email"><br/>
                         <label>Comment: </label><input type="text" name="content">
```

Comment:
I would use something like a text area here https://www.w3schools.com/tags/tag_textarea.asp since comments are usually multi line

Reply:
Changed comment to textarea.